package.json

```diff
@@ -16,7 +17,7 @@
     }
   },
   "scripts": {
-    "prebuild": "ts-node ./scripts/write-versions.ts",
+    "prebuild": "tsx ./scripts/write-versions.ts",
```

Comment:
ts-node struggles to run when package.json has `type: module`. Ran into this: 0962f516cb74b715d01434b21a295d27d7b09c60

Decided it was easier to swap to tsx https://www.npmjs.com/package/tsx

I might be missing something small... I didn't want to mess with the ts-config for the entire project just for this helper script.

Reply:
Ya probably better to use tsx? Seems like everything I can find online about it is positive

Reply:
@njooma Do you expect to have other more complex scripts that would benefit from types? If so, might be worth keeping tsx. If not, agreed that typescript isn't providing much value.

Reply:
Let's strip it for now and if we add more complex scripts then we can deal with it then. no need to keep it around for a "maybe" if it's entirely unused now (aside from this use case)

Reply:
Swapped to .js in 2fd1e38